src/test/java/net/openhft/chronicle/core/internal/JvmExceptionTracker.java

```diff
@@ -0,0 +1,93 @@
+package net.openhft.chronicle.core.internal;
+
+import net.openhft.chronicle.core.Jvm;
+import net.openhft.chronicle.core.onoes.ExceptionKey;
+import net.openhft.chronicle.core.onoes.Slf4jExceptionHandler;
+import net.openhft.chronicle.testframework.internal.ExceptionTracker;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+/**
+ * An implementation of ExceptionTracker that uses {@link Jvm} to track exceptions represented
+ * by {@link ExceptionKey}s.
+ */
+public final class JvmExceptionTracker implements ExceptionTracker<ExceptionKey> {
+    private final Map<Predicate<ExceptionKey>, String> ignoredExceptions = new LinkedHashMap<>();
+    private final Map<Predicate<ExceptionKey>, String> expectedExceptions = new LinkedHashMap<>();
+    private final Map<ExceptionKey, Integer> exceptions;
+
+    private JvmExceptionTracker(Map<ExceptionKey, Integer> exceptions) {
+        this.exceptions = exceptions;
+    }
+
+    /**
+     * Create a JvmExceptionTracker
+     *
+     * @return the exception tracker
+     */
+    public static JvmExceptionTracker create() {
+        return new JvmExceptionTracker(Jvm.recordExceptions());
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
JavaDocs are inherited by default so no need for `{@inheritDoc}` unless you add to the parent's method doc.